src/components/specific/files/folder-selector/FolderSelector.vue

```diff
@@ -65,7 +59,9 @@
           class="folder-selector__body__placeholder__icon"
           name="folderOpen"
           size="xxxl"
+          style="margin-bottom: 25px"
```

Comment:
We probably want static style in CSS instead of HTML. @LrxGaelle ?

Reply:
a set of margin in .scss for an icon is over-rided by the margin set in the design-system ! see below
![Screenshot 2021-11-16 at 10 26 52](https://user-images.githubusercontent.com/64323277/141958675-74bcf3a5-5e8a-4df7-ad26-b3fa486b5e1c.png)
![Screenshot 2021-11-16 at 10 30 10](https://user-images.githubusercontent.com/64323277/141959188-6ad2b847-e04a-4410-b46f-ba36cc808915.png)

Reply:
You can add `margin="0 0 25px 0px"` inside your `<BIMDataIcon>` component, instead of `style` prop

Reply:
thanks for the tips @LrxGaelle ! it's working well that way